zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -186,29 +191,39 @@ def get_entities(self, **kwargs) -> list:
             self.kube_client, self.cluster_id, self.alias, self.environment, self.region, self.infrastructure_account,
             namespace=self.namespace)
 
+        hpa_entities = get_cluster_hpas(
+            self.kube_client, self.cluster_id, self.alias, self.environment, self.region, self.infrastructure_account,
+            namespace=self.namespace
+        )
+
+        pcs_entities = get_cluster_credential_sets(
+            self.kube_client, self.cluster_id, self.alias, self.environment,
+            self.region, self.infrastructure_account, namespace=self.namespace
+        )
+
         postgresql_entities = []
         postgresql_cluster_entities = []
         postgresql_cluster_member_entities = []
         postgresql_database_entities = []
 
         try:
             if is_postgresql_operator_present(self.config_path):
-                    postgresql_entities = get_postgresqls(
-                        self.pg_client, self.cluster_id, self.alias, self.environment, self.region,
-                        self.infrastructure_account, namespace=self.namespace)
-
-                    postgresql_cluster_entities = get_postgresql_clusters(
-                        self.kube_client, self.cluster_id, self.alias, self.environment, self.region,
-                        self.infrastructure_account, self.hosted_zone_format_string, postgresql_entities,
-                        statefulset_entities, namespace=self.namespace)
-
-                    postgresql_cluster_member_entities = get_postgresql_cluster_members(
-                        self.kube_client, self.cluster_id, self.alias, self.environment, self.region,
-                        self.infrastructure_account, self.hosted_zone_format_string, namespace=self.namespace)
-
-                    postgresql_database_entities = get_postgresql_databases(
-                        self.cluster_id, self.alias, self.environment, self.region, self.infrastructure_account,
-                        self.postgres_user, self.postgres_pass, postgresql_cluster_entities)
+                postgresql_entities = get_postgresqls(
```

Comment:
This doesn't seem to be necessary for the PR.

Reply:
It is not but the formatting was wrong and my editor fixed it. I thought I would leave it in.